src/api/apis/MALAPI.ts

```diff
@@ -108,7 +108,7 @@ export class MALAPI extends APIModel {
 				url: result.url,
 				id: result.mal_id,
 
-				plot: result.synopsis,
+				plot: result.synopsis ? result.synopsis.replace('[Written by MAL Rewrite]', '').trim() : '',
```

Comment:
This and  line 177 isnt need on the MALAPI.ts, this is suppose to remove the end comment return from pulling plot synopsis. No all of them had it and this was something i personally add on my end, that ened up getting pushed to PR, this doesnt need to be added.

Reply:
this seems useful to keep